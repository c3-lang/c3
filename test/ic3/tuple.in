{[], []}
{"", ""}
{[], ""}
{"", []}
{a, b}
{a, b, c}
{a, b, c, d}
{a, b, c, d, e}
{a, b, c, d, e, f}
{a, b, c, d, e, f, g}
{a, b, c, d, e, f, g, h}
{a, b, c, d, e, f, g, h, i}
{a, b, c, d, e, f, g, h, i, j}
{{a, b}, {c, d}}
{{a, b}, {c, d}, {e, f}}
{{a, b}, {c, d}, {e, f}, {g, h}}
{{a, b}, {c, d}, {e, f}, {g, h}, {i, j}}

# comment 0
# comment 0b
{ # comment 1
  # comment 1b
  a # comment 2
    # comment 2b
  , # comment 3
    # comment 3b
  b ## comment 4
    ## comment 4b
  , ## comment 5
    ## comment 5b
  c ## comment 6
    ## comment 6b
  } ## comment 7
    ## comment 7b
