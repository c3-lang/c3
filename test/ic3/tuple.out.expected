{[], []}
{"", ""}
{"", []}
{[], ""}
{b, a}
{c, b, a}
{d, c, b, a}
{e, d, c, b, a}
{f, e, d, c, b, a}
{g, f, e, d, c, b, a}
{h, g, f, e, d, c, b, a}
{i, h, g, f, e, d, c, b, a}
{j, i, h, g, f, e, d, c, b, a}
{{d, c}, {b, a}}
{{f, e}, {d, c}, {b, a}}
{{h, g}, {f, e}, {d, c}, {b, a}}
{{j, i}, {h, g}, {f, e}, {d, c}, {b, a}}
